sim/hunter/mongoose_bite.go

```diff
@@ -0,0 +1,110 @@
+package hunter
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+	"github.com/wowsims/sod/sim/core/stats"
+)
+
+func (hunter *Hunter) getMongooseBiteConfig(rank int) core.SpellConfig {
+	spellId := [5]int32{0, 1495, 14269, 14270, 14271}[rank]
+	baseDamage := [5]float64{0, 25, 45, 75, 115}[rank]
+	manaCost := [5]float64{0, 30, 40, 50, 65}[rank]
+	level := [5]int{0, 16, 30, 44, 58}[rank]
+
+	raptorFuryDmgMult := 0.1
+	spellConfig := core.SpellConfig{
+		ActionID:      core.ActionID{SpellID: spellId},
+		SpellSchool:   core.SpellSchoolPhysical,
+		DefenseType:   core.DefenseTypeMelee,
+		ProcMask:      core.ProcMaskMeleeMHSpecial,
+		Flags:         core.SpellFlagMeleeMetrics | core.SpellFlagAPL,
+		Rank:          rank,
+		RequiredLevel: level,
+
+		ManaCost: core.ManaCostOptions{
+			FlatCost: manaCost,
+		},
+
+		Cast: core.CastConfig{
+			DefaultCast: core.Cast{
+				GCD: core.GCDDefault,
+			},
+			CD: core.Cooldown{
+				Timer:    hunter.NewTimer(),
+				Duration: time.Second * 5,
+			},
+		},
+
+		ExtraCastCondition: func(sim *core.Simulation, target *core.Unit) bool {
+			return hunter.DistanceFromTarget <= 5 && hunter.DefensiveState.IsActive()
+		},
+
+		BonusCritRating:  float64(hunter.Talents.SavageStrikes) * 10 * core.CritRatingPerCritChance,
+		CritDamageBonus: hunter.mortalShots(),
+		DamageMultiplier: 1,
+
+		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+			hunter.DefensiveState.Deactivate(sim)
+
+			multiplier := 1.0
+			if stacks := hunter.RaptorFuryAura.GetStacks(); stacks > 0 {
+				multiplier *= 1 + raptorFuryDmgMult*float64(stacks)
+			}
+			
+			damage := multiplier * (baseDamage + (hunter.GetStat(stats.AttackPower) * 0.4))
```

Comment:
`+ (hunter.GetStat(stats.AttackPower) * 0.4)`

Is this handling this bit?
> Mongoose Bite also deals additional damage equal to 40% of your Attack Power.

If so we should add it only the hunter has the Cobra Strikes rune equipped (part of separating the rune from the Mongoose Bite spell)